wideq/client.py

```diff
@@ -307,14 +318,18 @@ def load_model_info(self):
 
 class ModelInfo(object):
     """A description of a device model's capabilities.
-    """
-
+        """
+    
     def __init__(self, data):
         self.data = data
+    
+    @property
+    def model_type(self):
+        """ get model type"""
```

Comment:
What is the return type of this function? It would be useful to know what kind of data to expect from the API.